spec/units/peep_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'peep'
+require 'database_helpers'
+
+RSpec.describe Peep do
+  describe '.all' do
+    it 'returns all messages' do
+      peep = Peep.add(message: 'This is a peep!', date: '2022-07-15')
+      Peep.add(message: 'This is also a peep!', date: '2022-07-10')
+      Peep.add(message: 'This is a third peep!', date: '2022-07-05')
+      
+      peeps = Peep.all
+
+      expect(peeps.length).to eq 3
+      expect(peeps.first).to be_a Peep
+      expect(peeps.first.id).to eq peep.id
+      expect(peeps.first.message).to eq("This is a peep!")
+      expect(peeps.first.date).to eq("2022-07-15")
```

Comment:
Since the peeps array could have duplicates (due to a bug), it's also necessary to test that the other 2 peeps are the ones you expect.